tests/foreman/api/test_errata.py

```diff
@@ -925,3 +927,140 @@ def test_errata_installation_with_swidtags(
+   """
+
+
+@pytest.fixture(scope='module')
+def rh_repo_module_manifest(module_manifest_org):
+    """Use module manifest org, creates tools repo, syncs and returns RH repo."""
+    # enable rhel repo and return its ID
+    rh_repo_id = enable_rhrepo_and_fetchid(
+        basearch=DEFAULT_ARCHITECTURE,
+        org_id=module_manifest_org.id,
+        product=constants.PRDS['rhel8'],
+        repo=constants.REPOS['rhst8']['name'],
+        reposet=constants.REPOSET['rhst8'],
+        releasever='None',
+    )
+    # Sync step because repo is not synced by default
+    rh_repo = entities.Repository(id=rh_repo_id).read()
+    rh_repo.sync()
+    return rh_repo
+
+
+@pytest.fixture(scope='module')
+def rhel8_custom_repo_cv(module_manifest_org):
+    """Create repo and publish CV so that packages are in Library"""
+    return setup_org_for_a_custom_repo(
+        {
+            'url': settings.repos.module_stream_1.url,
+            'organization-id': module_manifest_org.id,
+        }
+    )
+
+
+@pytest.fixture(scope='module')
+def rhel8_module_ak(
+    module_manifest_org, default_lce, rh_repo_module_manifest, rhel8_custom_repo_cv
+):
+    rhel8_module_ak = entities.ActivationKey(
+        content_view=module_manifest_org.default_content_view,
+        environment=entities.LifecycleEnvironment(id=module_manifest_org.library.id),
+        organization=module_manifest_org,
+    ).create()
+    # Ensure tools repo is enabled in the activation key
+    rhel8_module_ak.content_override(
+        data={
+            'content_overrides': [{'content_label': constants.REPOS['rhst8']['id'], 'value': '1'}]
+        }
+    )
+    # Fetch available subscriptions
+    subs = entities.Subscription(organization=module_manifest_org).search(
+        query={'search': f'{DEFAULT_SUBSCRIPTION_NAME}'}
+    )
+    assert subs
+    # Add default subscription to activation key
+    rhel8_module_ak.add_subscriptions(data={'subscription_id': subs[0].id})
+    # Add custom subscription to activation key
+    product = entities.Product(organization=module_manifest_org).search(
+        query={'search': "redhat=false"}
+    )
+    custom_sub = entities.Subscription(organization=module_manifest_org).search(
+        query={'search': f"name={product[0].name}"}
+    )
+    rhel8_module_ak.add_subscriptions(data={'subscription_id': custom_sub[0].id})
+    return rhel8_module_ak
+
+
+@pytest.mark.tier2
+def test_apply_modular_errata_using_default_content_view(
+    module_manifest_org,
+    default_lce,
+    rhel8_contenthost,
+    rhel8_module_ak,
+    rhel8_custom_repo_cv,
+    default_sat,
+):
+    """
+    Registering a RHEL8 system to the default content view with no modules enabled results in
+    no modular errata or packages showing as applicable or installable
+
+    Enabling a module on a RHEL8 system assigned to the default content view and installing an
+    older package should result in the modular errata and package showing as applicable and
+    installable
+
+
```

Comment:
Consider removing the extra empty lines here.

Reply:
OK, trimmed those